discovery_engine_core/web-api/src/bin/ingestion.rs

```diff
@@ -354,8 +371,25 @@ async fn handle_rejection(err: Rejection) -> Result<impl Reply, Infallible> {
     let json = warp::reply::json(&ErrorMessage {
         code: code.as_u16(),
         message: message.into(),
+        log_id,
         errored_ids,
     });
 
-    Ok(warp::reply::with_status(json, code))
+    Ok(Box::new(warp::reply::with_status(json, code)))
+}
+
+fn setup_request_span(handler_name: &str) -> (Span, Uuid) {
+    //FIXME use v6/v7 once supported/standardized
```

Comment:
I don't think is particularly useful to have a fixme that we have no clue when it could be "fixed" and will bring none to minimal benefits. If we want something that has lex order we can use ulid and in any case that will not exempt us from having a timestamp in a more computer and human readable form.